Sources/Classes/core/Downloader/MiniAppClient.swift

```diff
@@ -264,10 +267,23 @@ internal class MiniAppClient: NSObject, URLSessionDownloadDelegate {
             delegate?.downloadFileTaskCompleted(url: "", error: NSError.downloadingFailed())
             return
         }
-        checkFileSignature(destinationURL: destinationURL, location: location)
+        fileDownloadingCompleted(destinationURL: destinationURL, at: location)
+    }
+
+    private func fileDownloadingCompleted(destinationURL: String, at location: URL) {
+        isSignatureVerified = false
+        isDownloadStatusUpdated = false
+        let ids = idsForUrls[destinationURL]
+        let tempPathDirName = (ids?.0 ?? "") + "/" + (ids?.1 ?? "")
+        guard let tempLocation = self.delegate?.moveFileToTempLocation(from: location, to: tempPathDirName) else {
+            delegate?.downloadFileTaskCompleted(url: "", error: NSError.downloadingFailed())
+            return
+        }
+        checkFileSignature(destinationURL: destinationURL, location: tempLocation)
     }
 
     private func checkFileSignature(destinationURL: String, location: URL) {
+        MiniAppLogger.d("LEO: checkFileSignature", destinationURL)
```

Comment:
do we still need this log? If yes I suggest something without `LEO` inside it ^^